pkg/ui/webpack.app.js

```diff
@@ -116,10 +123,24 @@ module.exports = (distDir, ...additionalRoots) => ({
 
   devServer: {
     contentBase: path.join(__dirname, distDir),
-    proxy: [{
-      context: ["/_admin", "/_status", "/ts", "/_auth"],
-      secure: false,
-      target: process.env.TARGET,
-    }],
+    index: "",
+    proxy: {
+      // Note: this shouldn't require a custom bypass function to work;
+      // docs say that setting `index: ''` is sufficient to proxy `/`.
+      // However, that did not work, and may require upgrading ty webpack 4.x.
+      "/": {
+        secure: false,
+        target: process.env.TARGET,
+        bypass: (req) => {
+          if (req.path === "/") {
```

Comment:
is it worth putting `"/"` in `proxyPrefixes`?

Reply:
Needs to be a special case because if it were treated as a prefix, it'd match everything. Will move logic to `shouldProxy` though.